sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapi/AADJwtGrantedAuthoritiesConverter.java

```diff
@@ -30,13 +34,16 @@
         for (String authority : getAuthorities(jwt)) {
             grantedAuthorities.add(new SimpleGrantedAuthority(authority));
         }
+        if (LOGGER.isDebugEnabled()) {
+            LOGGER.debug("User's authorities: {}.", grantedAuthorities);
```

Comment:
it seems too simple for a log, probably provide more context info? I'm not sure

Reply:
How about `User's authorities created from jwt token: {}.`